packages/create-llama/e2e/basic.spec.ts

```diff
@@ -91,17 +91,19 @@ for (const templateType of templateTypes) {
               test.skip(appType === "--no-frontend");
               await page.goto(`http://localhost:${port}`);
               await page.fill("form input", "hello");
-              await page.click("form button[type=submit]");
-              const response = await page.waitForResponse(
-                (res) => {
-                  return (
-                    res.url().includes("/api/chat") && res.status() === 200
-                  );
-                },
-                {
-                  timeout: 1000 * 60,
-                },
-              );
+              const [response] = await Promise.all([
+                page.waitForResponse(
+                  (res) => {
+                    return (
+                      res.url().includes("/api/chat") && res.status() === 200
+                    );
+                  },
+                  {
+                    timeout: 1000 * 60,
+                  },
+                ),
+                page.click("form button[type=submit]"),
```

Comment:
minor adjustment to fix random failure in e2e test. We should wait for response handling and click submit together.